docs/ConcurrencyRoadmap.md

```diff
@@ -0,0 +1,265 @@
+- Because this synchronization is now understood by the compiler, you cannot _forget_ to use the queue to protect state: the compiler will ensure that you are running on the queue in the class's methods, and it will prevent you from accessing the state outside those methods.
+- Because the compiler is responsible for doing this, it can be smarter about optimizing away synchronization, like when a method starts by calling an async function on a different actor.
+
+Having this static relationship between the actor and its functions and properties lets us enforce the isolation of data to an actor and define away data races. We statically know whether we’re in a context that can safely access an actor’s properties, and if not, the compiler handles switching into that context.
+
+Above we've shown an actor class, where you’ve got a tightly-encapsulated set of properties and code. But the way we do UI programming today often spreads code across a large number of classes that you’re supposed to use from a single main thread. That main thread is still a kind of actor — it’s what we call a global actor.
+
+You can mark classes and functions as being tied to that actor with an attribute. The compiler will let you reference this class from anywhere, but to actually call this method, you need to be on the UI actor. So, if it was appropriate for all the actions of `PlayerRefreshController` to be performed on the global UI actor, we would represent it like this:
+
+```swift
+@UIActor
+class PlayerRefreshController {
+  var players: [String] = []
+  var gameSession: GameSession
+    
+  func refreshPlayers() async {  ...  }
+}
+```
+
+## Proposals for the First Phase
+
+We will be pitching the following proposals in the coming weeks in support of the first phase:
+
+- **`async`/`await`** introduces a [coroutine-based](https://en.wikipedia.org/wiki/Coroutine) `async`/`await` model to Swift. Functions will opt into to being `async`, and can then `await` the results of other `async` functions, allowing asynchronous code to be expressed in a more natural "straight-line" form. Pitch [here](https://forums.swift.org/t/concurrency-asynchronous-functions/41619), proposal [here](https://github.com/DougGregor/swift-evolution/blob/async-await/proposals/nnnn-async-await.md).
+
+- **`Task` API and Structured Concurrency** introduces the concept of a task to the standard library. This will cover APIs to create detached tasks, task "nurseries" for dynamically creating child tasks,  and the mechanics for cancellation and prioritization of tasks. It also introduces scope-based mechanisms for awaiting values from multiple child tasks, based on the principles of [structured concurrency](https://en.wikipedia.org/wiki/Structured_concurrency). Pitch [here](https://forums.swift.org/t/concurrency-structured-concurrency/41622), proposal [here](https://github.com/DougGregor/swift-evolution/blob/structured-concurrency/proposals/nnnn-structured-concurrency.md).
```

Comment:
```suggestion
- **`Task` API and Structured Concurrency** introduces the concept of a task to the standard library. This will cover APIs to create detached tasks, task "nurseries" for dynamically creating child tasks, and the mechanics for cancellation and prioritization of tasks. It also introduces scope-based mechanisms for awaiting values from multiple child tasks, based on the principles of [structured concurrency](https://en.wikipedia.org/wiki/Structured_concurrency). Pitch [here](https://forums.swift.org/t/concurrency-structured-concurrency/41622), proposal [here](https://github.com/DougGregor/swift-evolution/blob/structured-concurrency/proposals/nnnn-structured-concurrency.md).
```